docs/l2-l3.rst

```diff
@@ -287,3 +293,15 @@ The application displays APIC VLAN tags for a particular Logical Device Cluster
 **Q. What is the best way to delete LDEV from APIC?**
 
 Do not delete Logical devices from APIC directly. Instead, as a first step, delete self IPs, VLANs and routes from the BIG-IP device by using the application. When you are done, you can delete the Logical Device from APIC. This ensures there are no stale self IP, VLAN, and route entries on the BIG-IP.
+
+-----
+
+**Q. Why do I see “Sync Failed” state on both the HA devices when Sync to BIG-IP button is clicked on L2-L3 Stitching Tab?**
```

Comment:
Why do I see a "Sync Failed" state on both the HA devices when clicking the Sync to BIG-IP button on the L2-L3 Stitching Tab?